WordPress/gutenberg
# Social Icons: add Doctolib service

## What?

This introduces a new Social Link icon, for the Doctolib service.

## Why?

Doctolib offers profile pages that health professionals can use to list their presence on the web.

## How?

This adds the new icon and service, as well as tests.

## Testing Instructions


1. Go to Posts > Add New
2. Add the Social Links block
3. Within the block, add a new Gravatar block
4. Publish your post
5. View on the frontend.

### Testing Instructions for Keyboard

No change

Note:I used https://github.com/WordPress/gutenberg/pull/56544 as a template and changed filenames and svg icons only, so everything should be functionally equivalent
![Capture d’écran du 2023-12-18 16-51-17](https://github.com/WordPress/gutenberg/assets/97767846/5355c664-3bc9-4e9f-9262-f723df1b56eb)
